tests/websocket.rs

```diff
@@ -502,7 +500,7 @@ async fn close_old_connections_on_limit() {
     assert_eq!(ws_for_apps, vec!["app_1", "app_3"]);
 }
 
-fn create_test_configuration(admin_port: u16) -> HcHttpGatewayResult<Configuration> {
+fn create_test_configuration(admin_port: u16) -> anyhow::Result<Configuration> {
```

Comment:
Nice, yes in tests it's helpful

Reply:
actually there's no ooint to return a result here at all if it's always unwrapped https://github.com/holochain/hc-http-gw/pull/25/commits/fbb829168a885f27395aadfd1d3f5fe961efa5f3